src/main/scala/beam/router/BeamSkimmer.scala

```diff
@@ -112,11 +126,10 @@ class BeamSkimmer @Inject()(
     origin: Location,
     destination: Location,
     departureTime: Int,
-    vehicleTypeId: org.matsim.api.core.v01.Id[BeamVehicleType],
-    beamServices: BeamServices
+    vehicleTypeId: org.matsim.api.core.v01.Id[BeamVehicleType]
   ): (Double, Double) = {
-    val origTaz = beamServices.tazTreeMap.getTAZ(origin.getX, origin.getY).tazId
-    val destTaz = beamServices.tazTreeMap.getTAZ(destination.getX, destination.getY).tazId
+    val origTaz = beamServicesOpt.get.tazTreeMap.getTAZ(origin.getX, origin.getY).tazId
```

Comment:
Could we more gracefully handle this and be explicit in the exception that will be thrown to help later debugging? Just calling get makes using options pointless as it results in the same problems as null pointers. A high level match could work here, just as it is elsewhere:

```
beamServicesOpt match {
  case None => throw MeaningfulException
  case Some(beamServices) => RestOfLogic
}
```